spread.yaml

```diff
@@ -284,6 +284,7 @@ restore: |
     fi
 
     rm -f $SPREAD_PATH/snapd-state.tar.gz
+    rm -rf $GOPATH/*
```

Comment:
there's one bug I've been fixing when i come across it, that so far isn't caught by shellcheck, and that is that `GOPATH` is a *path*; it's a series of directories separated by a `:`. I've been changing uses like the above for `${GOPATH%%:*}`, which prunes everything after the first `:`. Can you do that?